Extension/src/LanguageServer/client.ts

```diff
@@ -402,6 +402,7 @@ class DefaultClient implements Client {
         let serverModule: string = getLanguageServerFileName();
         let exeExists: boolean = fs.existsSync(serverModule);
         if (!exeExists) {
+            telemetry.logLanguageServerEvent("missingBinary");
```

Comment:
Do you want to log which binary name specifically is "missing"? In this case since it is files that we need that we are creating, it might be useful to log the filename only (no path) so you can see how often it is missing.

Reply:
This telemetry is only shown for the Microsoft.VSCode.CPP.Extension.exe -- that's the main scenario I'm interested in.